src/sidebar/util/build-thread.js

```diff
@@ -253,38 +227,63 @@ function hasVisibleChildren(thread) {
 }
 
 /**
- * @typedef Options
- * @prop {string[]} selected - List of currently-selected annotation ids, from
- *       the data store
+ * @typedef BuildThreadOptions
+ * @prop {Object.<string, boolean>} expanded - Map of thread id => expansion state
  * @prop {string[]} forcedVisible - List of $tags of annotations that have
  *       been explicitly expanded by the user, even if they don't
  *       match current filters
+ * @prop {string[]} selected - List of currently-selected annotation ids, from
+ *       the data store
  * @prop {(a: Annotation) => boolean} [filterFn] - Predicate function that
  *       returns `true` if annotation should be visible
  * @prop {(t: Thread) => boolean} [threadFilterFn] - Predicate function that
  *       returns `true` if the annotation should be included in the thread tree
- * @prop {Object.<string, boolean>} expanded - Map of thread id => expansion state
- * @prop {(a: Annotation, b: Annotation) => boolean} sortCompareFn - Less-than
- *       comparison function for sorting top-level annotations
- * @prop {(a: Annotation, b: Annotation) => boolean} replySortCompareFn - Less-than
- *       comparison function for sorting replies
+
+ * @prop {(a: Thread, b: Thread) => number} [sortCompareFn] - comparison
+ *       function for sorting top-level annotations
+ * @prop {(a: Thread, b: Thread) => number} [replySortCompareFn] - comparison
+ *       function for sorting replies
```

Comment:
It doesn't look like we actually override `replySortCompareFn` anywhere. We could just remove this option and make `defaultReplySortCompareFn` the only sorter.

Edit: I see that you mentioned it wasn't ever used anywhere below. If we can simplify things by just removing the code, let's do that.

Reply:
Removing. We can add back in if we find ourselves in need of different reply sorting.